ci/helm-chart/values.yaml

```
@@ -35,13 +35,12 @@ service:
 ingress:
   enabled: false
   #annotations:
-  #  kubernetes.io/ingress.class: nginx
   #  kubernetes.io/tls-acme: "true"
   #hosts:
   #  - host: code-server.example.loc
   #    paths:
   #      - /
-
+  ingressClassName:
```

add an empty string just to make it clearer that the field is intentionally empty:

```suggestion
  ingressClassName: ""
```

---

Whoops, thanks!